src/credentials/GroupDataProviderImpl.h

```diff
@@ -169,17 +179,16 @@ class GroupDataProviderImpl : public GroupDataProvider
                                   MutableByteSpan & ciphertext) const override;
         CHIP_ERROR DecryptMessage(const ByteSpan & ciphertext, const ByteSpan & aad, const ByteSpan & nonce, const ByteSpan & mic,
                                   MutableByteSpan & plaintext) const override;
-        CHIP_ERROR EncryptPrivacy(MutableByteSpan & header, uint16_t session_id, const ByteSpan & payload,
-                                  const ByteSpan & mic) const override;
-        CHIP_ERROR DecryptPrivacy(MutableByteSpan & header, uint16_t session_id, const ByteSpan & payload,
-                                  const ByteSpan & mic) const override;
+        CHIP_ERROR PrivacyEncrypt(const ByteSpan & input, const ByteSpan & nonce, MutableByteSpan & output) const override;
```

Comment:
Methods on this interface use the format (Verb)(Noun), if you invert the order, at least you should change the order of the other function to maintain consistency (MessageDecrypt instead of DecryptMessage).

Reply:
Renamed as advised.